Makefile

```diff
@@ -236,8 +236,9 @@ depend: init | $(ISTIO_OUT)
 $(ISTIO_OUT) $(ISTIO_BIN):
 	@mkdir -p $@
 
-$(ISTIO_OUT)/dep.png: $(ISTIO_OUT)/dep.dot
```

Comment:
BTW - why remove this target ???

Reply:
It's no longer being used.

Reply:
What do you mean 'no longer used' ? Did you send an email and asked if no developer is using it ? I do use it for example, and everyone wanting to get insight into the dep mess should use it as well.

Reply:
I'm now generating a dep.diff file, so it's different.  It serves the same purpose.

Reply:
How can the dep.diff generate a png file ?